tests/scripts/misc/radvd_slaac.conf.in

```diff
@@ -0,0 +1,12 @@
+interface WPAN { 
+        AdvSendAdvert on;
+        MinRtrAdvInterval 3; 
+        MaxRtrAdvInterval 10;
+        AdvManagedFlag off;
+        prefix 2001:db8:1:2::/64 { 
+                AdvOnLink on; 
+                AdvAutonomous on; 
+                AdvRouterAddr on; 
```

Comment:
```suggestion
                AdvRouterAddr off; 
```

Reply:
We are only listening on RA received on wpan0 interface.